vspace/src/main/java/edu/asu/diging/vspace/core/factory/impl/ChoiceFactory.java

```diff
@@ -0,0 +1,36 @@
+package edu.asu.diging.vspace.core.factory.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.vspace.core.factory.IChoiceFactory;
+import edu.asu.diging.vspace.core.model.IChoice;
+import edu.asu.diging.vspace.core.model.impl.Choice;
+import edu.asu.diging.vspace.core.services.ISequenceManager;
+
+@Service
+public class ChoiceFactory implements IChoiceFactory{
+    
+    @Autowired
+    private ISequenceManager sequenceManager;
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see
+     * edu.asu.diging.vspace.core.factory.impl.ISlideFactory#createChoices(java.util.List<java.lang.String>)
+     */
+    @Override
+    public List<IChoice> createChoices(List<String> choiceIds) {
+        List<IChoice> choices = new ArrayList<IChoice>();
+        for(String choiceId: choiceIds) {
+            IChoice choice = new Choice();
+            choice.setSequence(sequenceManager.getSequence(choiceId));
```

Comment:
Are choiceIds sequence ids? Or why do you call get sequence with a choice Id? Is so, very bad variable naming.